libs/s25main/WineLoader.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+///////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include "helpers/MultiArray.h"
+#include "ogl/glArchivItem_Bitmap.h"
+#include "ogl/glSmartBitmap.h"
+#include "ogl/glTexturePacker.h"
+#include "gameTypes/BuildingType.h"
+#include "gameTypes/GoodTypes.h"
+#include "gameTypes/JobTypes.h"
+#include "gameTypes/Nation.h"
+#include <map>
+
+// Wine Addon animation and images
+
+namespace wineaddon {
+
+struct BuildingImages
+{
+    unsigned building;
+    unsigned building_shadow;
+    unsigned buildingskeleton;
+    unsigned buildingskeleton_shadow;
+    unsigned door;
+    unsigned building_winter;
+    unsigned building_shadow_winter;
+    unsigned door_winter;
+    unsigned icon;
+};
+
+bool isWineAddonBuildingType(BuildingType bld);
+bool isWineAddonGoodType(GoodType good);
+bool isWineAddonJobType(Job job);
+
+extern std::map<std::pair<Nation, BuildingType>, BuildingImages> buildings;
+using BobEntryRange = std::pair<unsigned, unsigned>;
+struct BobEntry
+{
+    BobEntryRange bobEntryRange;
+    std::string description;
+};
+
+enum BobTypes
```

Comment:
Please only typed enums:
```suggestion
enum class BobTypes
```